internal/widget/divider.go

```diff
@@ -13,20 +13,20 @@ var _ fyne.Widget = (*Divider)(nil)
 // Divider is a widget for displaying a divider with themeable color.
 type Divider struct {
 	Base
-	colorFn func() color.Color
-	minSize fyne.Size
+	ColorFn func() color.Color
+	MinSz   fyne.Size
```

Comment:
I appreciate that you can't use MinSize, but not sure about `Sz` would Min be OK?

Reply:
I think `Min` is too generic. You cannot understand it without having a look at the type.
I’m still not happy with the exported fields at all.
Maybe it is not that bad to have a `NewMenuItemSeparator` constructor for the `Divider` instead?
And maybe call it `NewMenuItemDivider` to make more clear that it’s just another `Divider`?

Reply:
If you really think they should be different then this approach sounnds good.